### ISSUE TITLE
Integration Action "extra_properties" argument is not expected here

### ISSUE THREAD

**Issue by notfol** [opsgenie/terraform-provider-opsgenie#152]
### Terraform Version
```
$ terraform -v
Terraform v0.12.26
+ provider.aws v2.69.0
+ provider.opsgenie v0.4.1
+ provider.template v2.1.2
```

### Affected Resource(s)
- opsgenie_integration_action

### Terraform Configuration Files
```hcl
resource "opsgenie_integration_action" "nagios" {
  ...
  create {
     name = "Create Service Alert"
     ...
     extra_properties = map(
       "Environment", "{{_payload.Environment}}",
        "Region", "{{_payload.AwsRegion}}"
     )
     ...
  }
  ...
}
```

### Debug Output
```
Error: Unsupported argument

  on nagios.tf line 44, in resource "opsgenie_integration_action" "nagios":
  44:     extra_properties = map(

An argument named "extra_properties" is not expected here.
```

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform init`
2. `terraform validate`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by notfol** [comment]
A question, perhaps unrelated. Does the `opsgenie_api_integration` resource support the `NagiosV2` api integration type? I get the error below. The [documentation](https://registry.terraform.io/providers/opsgenie/opsgenie/latest/docs/resources/api_integration) links a [page](https://docs.opsgenie.com/docs/integration-types-to-use-with-api) it claims is "The full list of options," which I interpret to be a complete list of supported api integration types. I can open a separate issue if needed.

```
resource "opsgenie_api_integration" "nagios_integrations" {
  for_each  = { for inputs in local.integration_inputs.nagios: inputs.index => inputs }

  name                  = each.value.data.name
  type                    = "NagiosV2"
  owner_team_id  = local.escalation_team_ids[each.value.data.owner_team_name]
  enabled              = each.value.data.enabled
  suppress_notifications  = each.value.data.suppress_notification
}
```

```
Error occurred with Status code: 422, Message: Either sendViaOEC or sendViaMarid should be selected., Took: 0.049000, RequestId: b949da46-f613-46f3-8493-218df9d74cc0
```

**Comment by ilteristabak** [comment]
This bug is fixed in #154, new release taken, 0.4.2
